build/broccoli/transpile-to-es5.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const babel = require('broccoli-babel-transpiler');
+
+/**
+ * Transpiles a tree of ES2015+ JavaScript files to ES5 with Babel.
+ */
+module.exports = function transpileToES5(inputNode, modules = false) {
```

Comment:
@chadhietala and I talked off-thread and I persuaded him it was not worth optimizing the AMD builds because they are only used in tests.